mods/ra/audio/voices.yaml

```
@@ -86,7 +86,7 @@ SpyVoice:
 ThiefVoice:
 	Voices:
 		Select: swhat1,syeah1
-		Action: saffirm1,smout1,sokay1
+		Action: saffirm1,smout1,sokay1,scandy,sclean1,ssteal1
```

```suggestion
		Move: saffirm1,smout1,sokay1
		Action: scandy,sclean1,ssteal1
```
like the `Spy` above.

---

That the `Mobile` trait has to change to use the `Move` voice as well, I suppose? (Not a problem, just something to set up.)

---

https://github.com/OpenRA/OpenRA/blob/e685731b337d76ba4d03b7dc84105029870236d4/mods/ra/rules/infantry.yaml#L652 still needs `Voice: Move`

---

Done.